### ISSUE TITLE
Feature request: enable support for "Pattern Overrides"

### ISSUE THREAD

**Issue by PascalClaro** [ndiego/icon-block#54]
It would add a lot of power and flexibility to the plugin if it could be used with the synced patterns overrides feature.

**Comment by ndiego** [comment]
Yup, I have already started working on this and will be available later this month before WordPress 6.7 is released.

**Comment by PascalClaro** [comment]
Awesome! BTW I love your plugins :) I’ve recently posted a video on Block Visibility https://youtu.be/k3corKkT6oU 😎 thank you for your great work!

**Comment by ndiego** [comment]
> Yup, I have already started working on this and will be available later this month before WordPress 6.7 is released.

I wrote this too quickly, I meant that [`contentOnly` editing support](https://github.com/ndiego/icon-block/issues/38) will be added in the next release. Unfortunately, synced pattern overrides are only available for a handful of Core WordPress blocks as of WordPress 6.7. There isn't an API for custom blocks to hook into at the moment. As soon as there is, I'll add support in the Icon Block.
